pepper-apis/src/main/java/org/broadinstitute/ddp/model/kit/KitZipCodeRule.java

```diff
@@ -0,0 +1,21 @@
+package org.broadinstitute.ddp.model.kit;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import org.jdbi.v3.core.Handle;
+
+public class KitZipCodeRule extends KitRule<String> {
+
+    private final Set<String> zipCodes;
+
+    public KitZipCodeRule(long id, Set<String> zipCodes) {
+        super(id, KitRuleType.ZIP_CODE);
+        this.zipCodes = new HashSet<>(zipCodes);
+    }
+
+    @Override
+    public boolean validate(Handle handle, String inputZipCode) {
+        return inputZipCode != null && zipCodes.contains(inputZipCode.trim());
+    }
```

Comment:
You will have a problem here with a lot (most?) of the zip codes that we collect being of the extended form, e.g.: 
**02461-1314**
This is a US thing.  You already associating the zip codes with countries, so maybe this is a US type of match?

Reply:
Ah, good point! Based on a cursory reading, it sounds like the last 4 digits are specific to addresses and mail delivery, and not to an area/region. I'm assuming that we just want the area zip code, so potential fix here is to strip out that second part from the input before matching?

Reply:
I think that would be fine. If we are confident in this place of the code that the user ZIPs have the proper `\d{5}(-\d4)?` format then you can just grab the first 5 chars and compare against our "supported zip code" list.